////////////////////////////////////////////////////////////////////////////////
//                                                                            //
//  Copyright (C) 2018, goatpig.                                              //
//  Distributed under the MIT license                                         //
//  See LICENSE-MIT or https://opensource.org/licenses/MIT                    //                                      
//                                                                            //
////////////////////////////////////////////////////////////////////////////////

#ifndef _H_BDV_CODEC_
#define _H_BDV_CODEC_

#include "protobuf/AddressBook.pb.h"
#include "protobuf/AddressData.pb.h"
#include "protobuf/CommonTypes.pb.h"
#include "protobuf/FeeEstimate.pb.h"
#include "protobuf/LedgerEntry.pb.h"
#include "protobuf/Utxo.pb.h"
#include "protobuf/NodeStatus.pb.h"
#include "protobuf/BDVCommand.pb.h"

#endif
